e2e/e2etests/e2etests.go

```diff
@@ -552,6 +545,15 @@ var AllE2ETests = []runner.E2ETest{
 		},
 		TestBitcoinStdMemoDepositAndCallRevertOtherAddress,
 	),
+	runner.NewE2ETest(
+		TestBitcoinStdMemoInscribedDepositAndCallName,
+		"deposit Bitcoin into ZEVM and call a contract with inscribed standard memo",
```

Comment:
What's the different with `bitcoin_std_memo_deposit_and_call`?

Reply:
The memo in this E2E is done by inscription. The existing one is don by OP_RETURN.